cmd/kogito/command_checks.go

```diff
@@ -2,9 +2,9 @@ package main
 
 import "fmt"
 
-// checkProjecLocally will verify if the project/namespace exists in the CLI context.
+// checkProjectLocally will verify if the project/namespace exists in the CLI context.
 // Won't fecth the cluster to verify if the project/namespace exists. This is a local validation only.
-func checkProjecLocally(project string) (localProject string, err error) {
+func checkProjectLocally(project string) (localProject string, err error) {
 	if len(project) == 0 {
 		if len(config.Namespace) == 0 {
 			return "", fmt.Errorf("Couldn't find any Project in the current context. Use 'kogito use-project NAME' to set the Kogito Project where the service will be deployed or pass '--project NAME' flag to this one")
```

Comment:
We don't need that. The log implementation that we have on CLI is the same for the Operator, which is zapper.

Also, the log output is redirected to the cli, so users will see this error message clearly in the console.

This is nice because we can use `log.Debug` for `-v` flag, `log.Error` or `fmt.Errorf` for error messages and `log.Info` for general output.

Reply:
ok.